exercises/mark-griffith/short-response.md

```diff
@@ -0,0 +1,23 @@
+# Unit 2 - Lesson 3: Introduction to JavaScript Functions
+## Short Response Questions
+
+1. What is the difference between function parameters and arguments?
+    
+     A Parameter is variable in the declaration of function, it is where an arguement would be stored.
+
+     An Argument is the actual value of this variable that gets passed to function.
+
+2. What happens when you invoke a JavaScript function with a missing argument? What about calling a function with _extra_ arguments?
+3. 
+     Invoking a JavaScript function with missing arguement would run but return either undefined since thats the value that is placed in its stance.
+     If extra arguements are passed JavaScript runs with the first one that is passed.
+
+3. Define _hoisting_. What impact does hoisting have on the way that we structure our JavaScript programs?
+    
+    Hoisting is the process of JavaScript lifting up funtions as they are invoked. 
```

Comment:
You're fully correct in that hoisting allows us to call a function before it's declared. However, hoisting does not occur when functions are invoked. When does hoisting occur?